README.md

```diff
@@ -0,0 +1,247 @@
+---
+permalink: /guides/use-appsody-cli/
+layout: guide-markdown
+title: Developing microservice applications with the CLI
+duration: 20 minutes
+releasedate: 2019-12-03
```

Comment:
not sure the purpose of this field, is there an updatedate: also?

Reply:
@alohr51 - I'm not aware of any YAML values for "date updated" - can you confirm please?